apps/qubit/modules/search/actions/advancedAction.class.php

```diff
@@ -450,6 +462,65 @@ static function addToQueryBool(&$queryBool, $operator, $queryField)
       }
   }
 
+  static function getDateRangeQuery($start, $end)
+  {
+    if (empty($start) && empty($end))
+    {
+      return;
+    }
+
+    $query = new \Elastica\Query\Bool();
+    $range = array();
+
+    if (!empty($start))
```

Comment:
Maybe we could add && empty($end) to the condition here, and likewise && empty($start) in the second if() below, that way it isn't assigning all these variables when it doesn't need to, if I've understood the code correctly